### ISSUE TITLE
WorldGuard checklist

### ISSUE THREAD

**Issue by wizjany** [SpongePowered/SpongeAPI#812]
## API
- [x] [refactor/event-names](https://github.com/SpongePowered/SpongeAPI/tree/refactor/event-names) branch
 - [x] PLUGIN/UNKNOWN SpawnType
- [ ] Other
 - [ ] Way to get all registered (or otherwise) services from the service manager

## Impl
- [ ] Data Implementation (see also: SpongePowered/SpongeCommon#8)
 - [x] Vehicle/PassengerData (currently undergoing renovations)
 - [x] TameableData
 - [x] HealthData
 - [x] FoodData
 - [x] BreathableData
 - [x] IgniteableData
 - [x] VelocityData
 - [x] SignData
 - [ ] PotionEffectData
 - [x] GameModeData
- [ ] Event implementation
 - [ ] Will starting testing once api is done

**Comment by hsyyid** [comment]
I'll do `Igniteable Data`

**Comment by Aaron1011** [comment]
I'll take `PotionEffectData`

**Comment by gabizou** [comment]
If you are claiming to implement a `DataProcessor`, please also note it on the SpongePowered/SpongeCommon#8 issue.

**Comment by JBYoshi** [comment]
> Way to get all registered (or otherwise) services from the service manager

Why?

**Comment by Zidane** [comment]
Closing this as all of this has been covered and if something is missing then it needs to be its own issue.
